app/Traits/ValuesToArrayTrait.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Traits;
+
+trait ValuesToArrayTrait
```

Comment:
Trafiłem ostatnio na to samo, enum nie może po niczym dziedziczyć, trait działa? Nie wpadłem na to :D najs

Reply:
testy przechodzą, więc musi działać :) 
https://www.php.net/manual/en/language.enumerations.traits.php